### PR TITLE
Get release 5 ready - Swift 4 support

### PR DESCRIPTION
**Pull request by JakeLin**
Get release 5 ready and support Swift 4 💥 

Related to #513

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,8 +31,10 @@ With `IBAnimatable`, we can design a UI in Interface Builder like what we can do
 
 As a designer, we love Sketch, which is a simple but yet super powerful tool to create UI. However, Sketch can't design interaction, navigation, transition and animation, and we may need another tool like Framer to design some of them. Moreover, to make an App Store ready App, we need to use Xcode and Interface Builder to implement the UI and animations. To speed up the process and minimize the waste, we create `IBAnimatable` to make Interface Builder designable and animatable.   
 
-## Version 4
-IBAnimatable 4.0 is the latest major release of IBAnimatable. This version introduces promise-like animation API, and contains some breaking changes from version 3.x. Please check out [IBAnimatable 4.0 Migration Guide](Documentation/IBAnimatable 4.0 Migration Guide.md) for more information about how to migrate your project to 4.0. 
+## Version 5
+IBAnimatable 5.0 is the latest major release of IBAnimatable. This version supports Swift 4, there is not API breaking changes from migrating from version 4.x.
```
Comment (gkye) [inline README.md]: > there is not API breaking changes 

Could be replace with "There are no API breaking changes"?

Comment (gkye) [reply README.md]: Everything else looks good

**Comment by JakeLin** [comment]
@gkye thanks, have fixed the wording.

**Review by gkye — APPROVED** [review]

**Comment by gkye** [comment]
🙈

**Pull request merged**